package.json

```diff
@@ -45,30 +45,31 @@
   ],
   "preferGlobal": true,
   "dependencies": {
-    "chalk": "^4.1.2",
-    "commander": "^7.2.0",
-    "fs-extra": "^10.1.0",
-    "hap-nodejs": "~0.11.1",
-    "qrcode-terminal": "^0.12.0",
-    "semver": "^7.5.4",
-    "source-map-support": "^0.5.21"
+    "chalk": "4.1.2",
+    "commander": "12.0.0",
```

Comment:
I remember that we didn’t upgrade commander in a while as it broke some functionality as they change some of their APIs. Did we verify that the Homebridge CLI still works as expected? I think we do not have automated tests for that as of right now?

Reply:
Thank you for verifying this 👍